Production_scripts/Process_extracts/Produce_extract_tests/extract_homelessness_tests.R

```diff
@@ -0,0 +1,39 @@
+####################################################
+# Name of file - extract_homelessness_tests.R
+# Original Authors - Bateman McBride
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               homelessness processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Homelessness", ext = "rds")) %>%
+  dplyr::mutate(smrtype = stringr::str_trim(smrtype, "both"))
```

Comment:
```suggestion
new_data <- readr::read_rds(get_source_extract_path(year, "Homelessness")) 
```
Suggested to remove `ext = "rds"`

Do we need the mutate for smrtype here? I think this was applied to your version of the tests, but i dont think we use smrtype flags for counting in the function

Reply:
Suggestion implemented